src/Sentry/IHub.cs

```diff
@@ -124,4 +124,12 @@ public TransactionContext ContinueTrace(
     /// <param name="configureScope">Callback method to configure the scope.</param>
     /// <param name="hint">An optional hint providing high level context for the source of the event, including attachments</param>
     public void CaptureFeedback(SentryFeedback feedback, Action<Scope> configureScope, SentryHint? hint = null);
+
+    /// <summary>
+    /// Captures an attachment for an existing event.
+    /// </summary>
+    /// <param name="eventId">The ID of the event to add the attachment to.</param>
+    /// <param name="attachment">The attachment.</param>
+    /// <returns>true if the enveloped was queued, false otherwise.</returns>
+    public bool CaptureAttachment(SentryId eventId, SentryAttachment attachment);
```

Comment:
How do we deal with this? An addition to a public interface, but not added to `SentrySdk`. I don't even know what kind of changelog entry this should get. 😅 

Of course we could just make it public and then it's a feature?

Reply:
We have tests that explicitly check `SentrySdk` implements everything from `IHub`... so if it has to go on IHub then it's public basically. 

The other option would be to have it on the Hub only (as an internal member) and in the calling code you do something like:
```csharp
var hub = GetSomeIHubImplementationFromSomewhere();
if (hub is Hub realHub)
{
    hub.CaptureAttachment(foo, bar);
}
```